DOCUMENTATION.md

```diff
@@ -93,7 +93,9 @@ The repo detail view also provides features for performing the following git ope
 - Fetching changes from remote
```

Comment:
New directory navigator feature has been included which is not covered in the list